demo/PPConfig_test.ini

```diff
@@ -5,7 +5,7 @@
 
 # The simulation used for the ephemeris input.
 # Options: "ar", "external"
-ephemerides_type = ar
+ephemerides_type = external
```

Comment:
@mschwamb This is one of the config files that is rendered in the documentation. As you can see, I've changed the ephem_type here. Please let me know if this is a problem. 

Part of the reason for doing this was that the end to end test was failing when ephem_type = 'ar'. (Which makes sense, because the e2e test was created before A&R was built in)

Reply:
That's fine. I'll update the documentation to use sorcha_demo.ini instead. Sorry, I thought the end to end test is running in CI and was using a different config file